linkerd/core/src/main/scala/io/buoyant/linkerd/IdentifierInitializer.scala

```diff
@@ -0,0 +1,5 @@
+package io.buoyant.linkerd
+
+import io.buoyant.linkerd.config.ConfigInitializer
+
+trait IdentifierInitializer extends ConfigInitializer
```

Comment:
nbd but files should have newlines at the end.  maybe update your ide/editor to do that by default.